CHANGELOG.md

```diff
@@ -1,4 +1,83 @@
-## 3.44.0 (Unreleased)
+## 3.45.0 (Unreleased)
+
+UNKNOWN CHANGELOG TYPE:
+* Add iam policy resources for netbook instance ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
+* Add taxonomy and policy_tag resources to data_catalog product ([#2626](https://github.com/hashicorp/terraform-provider-google-beta/pull/2626))
```

Comment:
Should these replace the corresponding `New Resource` lines under the Features section? Otherwise we could just remove this block

Reply:
Just forgot to delete these lines!